VideoApp/VideoApp/Storyboards/Base.lproj/Main.storyboard

```diff
@@ -528,7 +528,7 @@
                                         </constraints>
                                     </view>
                                     <collectionView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" showsHorizontalScrollIndicator="NO" showsVerticalScrollIndicator="NO" dataMode="prototypes" translatesAutoresizingMaskIntoConstraints="NO" id="wU6-sZ-Ash">
-                                        <rect key="frame" x="10" y="690" width="404" height="172"/>
+                                        <rect key="frame" x="0.0" y="690" width="414" height="172"/>
```

Comment:
The UICollectionView had a gap on the left side of 10 points on the leading edge but was flush to the trailing edge. I think this was just an unintended bug (it's not obvious until you have more than a screen full of cells).

Reply:
I think I set this gap intentionally but not sure what desired UX really is to be honest. I'm good with this adjustment. 

Probably how it should really work someday is the collection view has no gaps on either side and instead we use spacing inside of the collection view content at start and end. This way it looks good at start/end position and when scrolling (cells are not clipped before the edge of the display). 

But I think the team wants to completely redo UX of this screen someday so best to tackle then.